src/oci-umount.c

```diff
@@ -430,6 +457,81 @@ static int is_mount_duplicate(const char*path, struct host_mount_info *mounts_on
 	return 0;
 }
 
+static int parse_options_section_line(const char *id, const char *conf_file, char *line) {
+	int str_len, i;
+	char *level_str;
+
+	str_len = strlen("log_level=");
+
+	if (strncmp(line, "log_level=", str_len)) {
```

Comment:
You are not handling

   log_level=
log_level = foobar

Reply:
should we really support whitespaces around `=`. I am not a big fan of that.

W.r.t `log_level=`, we are getting following error message.

`oci-umount[3906]: umounthook <error>: 10924c826513: Unknown log level  in file /etc/oci-umount/oci-umount-options.conf: Success`

Do you want to display anything else?

Reply:
If I am a human being and a computer complains about a space, I get pissed off.  Computers should handle name value pairs with spaces.

Reply:
So where are do you want to support spacing. I find this to be a weak argument. 

- How about section name. `[     options      ]` is it a valid string?
- How about `log_level                 =                   foo`. is it valid
- How about `log_level \t\t = \t\t foo`

I mean there are so many possibilities. And I don't see a strong argument to support those. How hard it is to type, `name=value`. This is so common.

Reply:
github ate away multiple spaces I had put. So we need to support multiple white spaces around `=` as well. And same is true for section  name? 

Does key name has to start from beginning of line? or we need to support white space there to.

Reply:
Yes I just want spaces trimmed from beginning and end of value.  Nothing too fancy

"log_level = debug  "
Should work.

Reply:
I would trim spaces in name and value.